Corzbank.Data/Migrations/20220504130809_addConstraints.cs

```diff
@@ -0,0 +1,218 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Corzbank.Data.Migrations
+{
+    public partial class addConstraints : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Why all the tables is in plural? Check how to rename those

Reply:
I think that's ok because in our context everything is in plural